<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRange</class>
 <widget class="QDialog" name="CreateRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Range</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic Range Selection</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="basicAddARangeButtonGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Add a Range of Pages</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0" colspan="5">
           <widget class="QRadioButton" name="basicConsecutiveRadioButton">
            <property name="text">
             <string>Consecutive Pages</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ScrSpinBox" name="basicConsecutiveFromSpinBox"/>
          </item>
          <item row="3" column="0" colspan="5">
           <widget class="QLineEdit" name="basicCommaSepLineEdit"/>
          </item>
          <item row="5" column="0" colspan="5">
           <widget class="QRadioButton" name="basicOddRadioButton">
            <property name="text">
             <string>Odd Pages</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="ScrSpinBox" name="basicConsecutiveToSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="basicConsecutiveFromLabel">
            <property name="text">
             <string>From:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="5">
           <layout class="QHBoxLayout">
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>38</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="basicRangeAddButton">
              <property name="text">
               <string>&amp;Add To Range</string>
              </property>
              <property name="shortcut">
               <string>Alt+A</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>38</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="4" column="0" colspan="5">
           <widget class="QRadioButton" name="basicEvenRadioButton">
            <property name="text">
             <string>Even Pages</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="basicConsecutiveToLabel">
            <property name="text">
             <string>To:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="QRadioButton" name="basicCommaSepRadioButton">
            <property name="text">
             <string>Comma Separated List</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox1">
         <property name="title">
          <string>Range of Pages</string>
         </property>
         <layout class="QGridLayout">
          <item row="2" column="1">
           <widget class="QPushButton" name="basicRangeDelButton">
            <property name="text">
             <string>De&amp;lete</string>
            </property>
            <property name="shortcut">
             <string>Alt+L</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="basicRangeDownButton">
            <property name="text">
             <string>Move &amp;Down</string>
            </property>
            <property name="shortcut">
             <string>Alt+D</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="basicRangeUpButton">
            <property name="text">
             <string>Move &amp;Up</string>
            </property>
            <property name="shortcut">
             <string>Alt+U</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="4">
           <widget class="QListWidget" name="basicRangeListBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced Reordering</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="2" column="0" colspan="5">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="advSampleOrderExampleLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="advSampleOrderLabel">
         <property name="text">
          <string>Sample Page Order:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="ScrSpinBox" name="advPageGroupSizeSpinBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="advPageGroupSizeLabel">
         <property name="text">
          <string>Page Group Size:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="pageCountLabel">
       <property name="text">
        <string>Number of Pages in Document:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pageCountValueLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
